test/interpret.jl

```diff
@@ -646,16 +645,16 @@ module VecTest
             Core.getfield(Base, :llvmcall)($exp, Vec{$N, $T}, Tuple{Vec{$N, $T}, Vec{$N, $T}}, x, y)
         end
     end
-    f() = 1.0 * one(Tensor{2,3})
+    f(a) = vecadd(a, a)
```

Comment:
If this "vendors" the implementation in Tensors, I'm all in favor. CC @KristofferC to check whether there was any more specific intent for the original version.

(We should vendor more of the tests and depend less on external packages.)

Reply:
I don't really recall the original intent. I guess best way would be to check where the test was intorudced and see that the new one also fails/succeeds in the same way.